src/arrayUtils.js

```diff
@@ -0,0 +1,68 @@
+if (typeof maybeExport === 'undefined') {
```

Comment:
oh, i see, so this is a hack that makes the `{{{` lines no-ops in HTML form? rather clever :) but couldn't we just move the `{{{` lines so they are printed from `jsifier`, right after it prints `arrayUtils.js`? It could literally do `print(maybeExport(..))` for each one, I think.